src/optimization/ControlFlow.cpp

```diff
@@ -1091,6 +1091,8 @@ void optimizations::addStartStopSegment(const Module& module, Method& method, co
 
 void optimizations::removeConstantLoadInLoops(const Module& module, Method& method, const Configuration& config)
 {
+    logging::debug() << "moveConstantsDepth = " << config.additionalOptions.moveConstantsDepth << logging::endl;
```

Comment:
Is this parameter used anywhere? I didn't find a usage.

Reply:
I will implement function that users can specify a depth. Operations which in blocks whose distance from the most inner loop is less than this depth will be moved.

For example, when user specify `--fmove-constants-depth=0`, only operations in the most inner loop will be moved to one-outer block.